### PR TITLE
Adding Scrabble3D

### PR DESCRIPTION
**Pull request by DrStrangepork**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by DrStrangepork** [comment]
The SourceForge URL format as suggested by the documentation link does not work for this download.

**Review by miccal — APPROVED** [review]

File: Casks/scrabble3d.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask 'scrabble3d' do
+  version :latest
+  sha256 :no_check
+
+  # sourceforge.net/projects/scrabble was verified as official when first introduced to the cask
+  url 'https://sourceforge.net/projects/scrabble/files/latest/download'
```
Comment (vitorgalvao) [inline Casks/scrabble3d.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md), and with that a `version` and `sha256`.

File: Casks/scrabble3d.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+cask 'scrabble3d' do
+  version :latest
+  sha256 :no_check
+
+  # sourceforge.net/projects/scrabble was verified as official when first introduced to the cask
+  url 'https://downloads.sourceforge.net/scrabble/Scrabble3D-darwin.pkg'
+  name 'Scrabble3D'
```
Comment (vitorgalvao) [inline Casks/scrabble3d.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md), and with that a `version` and `sha256`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/scrabble3d.rb (hunk 2 above)
Comment (miccal) [inline Casks/scrabble3d.rb]: @vitorgalvao I cannot seem to find a valid `version` for this?

Comment (miccal) [inline Casks/scrabble3d.rb]: @vitorgalvao found the version in by expanding the `pkg` and viewing the file `PackageInfo`.

**Review by miccal — APPROVED** [review]

File: Casks/scrabble3d.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/scrabble3d.rb]: @miccal Nice one!

**Pull request merged**